apps/server/src/modules/h5p-editor/s3-config.ts

```diff
@@ -0,0 +1,10 @@
+import { Configuration } from '@hpi-schul-cloud/commons';
+import { S3Config } from '../files-storage/interface';
+
+export const s3ConfigLibraries: S3Config = {
+	endpoint: Configuration.get('H5P_EDITOR__S3_ENDPOINT') as string,
+	region: Configuration.get('H5P_EDITOR__S3_REGION') as string,
+	bucket: Configuration.get('H5P_EDITOR__S3_BUCKET_LIBRARIES') as string,
+	accessKeyId: Configuration.get('H5P_EDITOR__S3_ACCESS_KEY_ID') as string,
+	secretAccessKey: Configuration.get('H5P_EDITOR__S3_SECRET_ACCESS_KEY') as string,
```

Comment:
you shoud add this config keys to `config/default.schema.json`

Reply:
it's more like sync it with the keys in `default.schema.json`
https://github.com/hpi-schul-cloud/schulcloud-server/blob/THR-26-library-storage-s3/config/default.schema.json#L304-L342